tests/playwright/sanity/modules/floating-buttons/floating-elements-admin.test.ts

```diff
@@ -0,0 +1,209 @@
+import { parallelTest as test } from '../../../parallelTest';
+import FloatingElementPage from './floating-element-page';
+import { expect } from '@playwright/test';
+import EditorSelectors from '../../../selectors/editor-selectors';
+
+test.describe( 'Verify floating buttons editor, admin page and front page behavior', () => {
+	test.afterEach( async ( { browser, apiRequests }, testInfo ) => {
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const floatingElPage = new FloatingElementPage( page, testInfo, apiRequests );
+		await floatingElPage.deleteAllFloatingButtons();
+		await page.close();
+	} );
+
+	test( 'Verify editor behavior by creating post through API and also FE behavior', async (
+		{
+			browser,
+			page,
+			apiRequests,
+		}, testInfo ) => {
+		const floatingElPage = new FloatingElementPage( page, testInfo, apiRequests );
+		const editor = await floatingElPage.createFloatingButtonWithAPI();
```

Comment:
Are we testing both the API import method and the Elementor Editor method?

I think just testing the Editor is sufficient.

Reply:
@hein-obox floating elements only come through the remote library